packages/@icon-magic/generate/test/fixtures/out/home-2/ads/active-small-on-dark.svg

```diff
@@ -1,3 +1,4 @@
-<svg xmlns="http://www.w3.org/2000/svg" width="24" height="24" viewBox="0 0 24 24" class="flip" id="ads-active-small-on-dark" aria-hidden="true" role="none" data-supported-dps="24x24" fill="currentColor">
```

Comment:
Just checking about this change, is width, height and class flip removed supposed to happen?

Reply:
So that is because of how the test is structured. If you look at generate/index-test.js, what's happening is that we're running generate with and without hashing off. And the way to check is by changing the source SVG to one with and without the widths and the flip class.